shared/login/signup/invite-code.render.native.js

```diff
@@ -56,7 +56,8 @@ const styles = {
   },
   input: {
     alignSelf: 'stretch',
-    marginTop: 0
+    marginTop: 0,
+    flex: 0
```

Comment:
i think we should actually remove flex from the common component. Originally i did this with button fullWidth but ultimately removed it. its just too surprising since its driven by the parent.